test/test_torch.py

```diff
@@ -11324,6 +11324,19 @@ def test_c10_layer_norm(self):
                 weight), torch.tensor(bias), 1, epsilon, True)
         torch.testing.assert_allclose(expected_norm, actual_norm)
 
+    def test_subclass_tensors(self):
+        # raise an error when trying to subclass FloatTensor
+        with self.assertRaisesRegex(TypeError, "type 'torch.FloatTensor' is not an acceptable base type"):
```

Comment:
I'm curious if this error message is actually stable -- I guess the CI will tell us.